test/data/manifests/fedora-boot.mpp.json

```diff
@@ -70,7 +70,7 @@
           "root_fs_uuid": "76a22bf4-f153-4541-b6c7-0332c0dfaeac",
           "kernel_opts": "ro biosdevname=0 net.ifnames=0",
           "legacy": "i386-pc",
-          "saved_entry": "ffffffffffffffffffffffffffffffff-5.11.12-300.fc34.x86_64"
+          "saved_entry": {"mpp-format-string": "ffffffffffffffffffffffffffffffff-{rpms[\"stages\"][\"kernel-core\"][12:-4]}" }
```

Comment:
Single quotes should work here, no?

```suggestion
          "saved_entry": {"mpp-format-string": "ffffffffffffffffffffffffffffffff-{rpms['stages']['kernel-core'][12:-4]}" }
```

Reply:
You could just cherry-pick 226a707bf72cd42e12012248a0f4ce85fc57d474 to avoid `[12:-4]`.

Reply:
Hmm, the ' quotes don't work, because they termingate the  `f'` string we compose to eval... I wonder if we can fix that.

Reply:
For the cherry pick, lets do that when that lands (or here if that lands first)